spec/logSearch_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rspec'
+require_relative '../lib/logSearch'
```

Comment:
The spec_helper should be required here, the logSearch should be added to the spec_helper.